### PR TITLE
chore: document knapsack to knapsack_pro

### PR DESCRIPTION
**Pull request by 3v0k4**

### CODE REVIEW

File: MIGRATE_TO_KNAPSACK_PRO.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Migration steps: from Knapsack to Knapsack Pro
+
+Follow these steps to migrate from `knapsack` to `knapsack_pro` in 10 minutes.
+
+Commands are provided to help you with each step.
+
+On Linux, you need to remove the `''` part from the `sed` commands. Also, you can ignore the `sed: no input files` warning that is printed when there's no substitution to perform.
```
Comment (ArturT) [inline MIGRATE_TO_KNAPSACK_PRO.md:7]:
```suggestion
> [!TIP]
> On Linux, you need to remove the `''` part from the `sed` commands. Also, you can ignore the `sed: no input files` warning that is printed when there's no substitution to perform.
```

to get a tip look:
image

source: https://stackoverflow.com/a/72327818/905697

File: spec/knapsack/adapters/rspec_adapter_spec.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+require 'ostruct'
```
Comment (ArturT) [inline spec/knapsack/adapters/rspec_adapter_spec.rb:1]: Why do we need this? Was something failing because ostruct was not loaded?

**Review by ArturT — APPROVED** [review]
Awesome new look of the readme. I like migration steps. 🚀

File: MIGRATE_TO_KNAPSACK_PRO.md (hunk 1 above)
Comment (3v0k4) [reply MIGRATE_TO_KNAPSACK_PRO.md:7]: Oh. I didn't know about that!

File: spec/knapsack/adapters/rspec_adapter_spec.rb (hunk 2 above)
Comment (3v0k4) [reply spec/knapsack/adapters/rspec_adapter_spec.rb:1]: Tests for Ruby v2.3 [started failing](https://github.com/KnapsackPro/knapsack/actions/runs/9401495650/job/25893400671).

I didn't investigate it much, but maybe a change in a patch version (2.3.x)?

**Pull request merged**